docs/root/configuration/http/http_filters/api_key_auth_filter.rst

```diff
@@ -56,10 +56,12 @@ An example scope specific configuration of the filter may look like the followin
         route: { cluster: some_service }
         typed_per_filter_config:
           api_key_auth:
-            "@type": type.googleapis.com/envoy.extensions.filters.http.api_key_auth.v3.ApiKeyAuthPerScope
+            "@type": type.googleapis.com/envoy.extensions.filters.http.api_key_auth.v3.ApiKeyAuthPerRoute
```

Comment:
@wbpcode can we move this config to a bootstrap with literalinclude

Reply:
I can have a try

Reply:
if you do:

```
git grep literalinclude docs/
```

you should find lots of examples - we probs want to include a caption with download